library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,34 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for a dependency tree to contain conflicts.
+Unfortunately, as soon as any library in a dependency tree introduces a breaking
+change, consumers can't use the latest versions of those libraries together. To
```

Comment:
"those libraries" is unclear. I guess it means the libraries in the dependency tree?

Reply:
Yes. I will qualify it better.